lib/resource-providers/vpc.ts

```diff
@@ -42,4 +71,16 @@ export class DirectVpcProvider implements ResourceProvider<ec2.IVpc> {
     provide(_context: ResourceContext): ec2.IVpc {
         return this.vpc;
     }    
-}
\ No newline at end of file
+}
+
+/**
+ * Direct import secondary subnet provider, based on a known subnet ID. 
+ * Recommended method if secondary subnet id is known, as it avoids extra look-ups.
+ */
+export class LookupSubnetProvider implements ResourceProvider<ISubnet> {
+    constructor(private subnetId: string, private id?: string) { }
```

Comment:
private id?  - what is this for? is it used?

Reply:
Yes, fixed.